magento/inventory
# Remove "Ship" functionality for store pickup orders.

### Description (*)


### Fixed Issues (if relevant)

1. https://github.com/magento/inventory/issues/2568: Remove "Ship" functionality for store pickup orders
2. https://github.com/magento/inventory/issues/2160: Send Store Pickup confirmation email asynchronously

### Manual testing scenarios (*)

1. Create order with store pickup delivery method.
2. Try to create shipment via API rest/V1/order/:orderId/ship with body
{
  "items": [
    {
      "extension_attributes": {},
      "order_item_id": int,
      "qty": int
    }
  ],
  "notify": true,
  "appendComment": true,
  "comment": {
    "extension_attributes": {},
    "comment": "string",
    "is_visible_on_front": 0
  },
  "tracks": [  ],
  "packages": [  ],
  "arguments": {
    "extension_attributes": {
      "source_code": "string"
    }
  }
}
3. Verify, there is exception and shipment is not created
4. Try to notify order for pickup via API /V1/order/notify-orders-are-ready-for-pickup
{
    orderIds: [your_order_id]
}
5. Verify, shipment has been created, customer has been notified.

### Manual testing scenarios (*)
1. see https://github.com/magento/inventory/issues/2160

### Questions or comments


### Contribution checklist (*)
